docs/DeveloperGuide.md

```diff
@@ -269,13 +270,13 @@ _{Explain here how the data archiving feature will be implemented}_
 
 **Target user profile**:
 
-* has a need to manage a significant number of contacts
+* has a need to manage a significant number of volunteers and events
```

Comment:
I think shd change `events` to `volunteering events`

Reply:
good idea